README.md

```diff
@@ -206,6 +218,36 @@ The rule of thumb is:
   project, there is a directory at the root called `config`, and this will contain files like
   `config/conf.d/web-routes.conf`.
 
+### Bootstrap Files
+There are two ways of specifying a set of services to bootstrap your TK application with:
+
+1. A single `bootstrap.cfg` file in your project's `:config-dir` directory
+2. Two `services.d` directories which can contain any number of `*.cfg` files that will be
+   merged together
+
+The default is to use a single `bootstrap.cfg` directly under your `:config-dir`. Setting
```

Comment:
Would also mention the default value of the `:bootstrap-source` setting here, in case anyone prefers to be explicit in their config.